.github/workflows/test.yml

```diff
@@ -1,6 +1,16 @@
 name: Run tests against changes
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+      - '[0-9]+'
+  push:
+    branches:
```

Comment:
This looks fine. Should we add a `tags: [*]` as well?

Reply:
If we want to run CI on tag pushes aswell, then yes, of course!

Reply:
Perfect.